tools/ovc/openvino/tools/ovc/convert.py

```diff
@@ -72,7 +72,10 @@ def convert_model(
             do not add :0 to this name. The order of outputs in converted model is the
             same as order of specified operation names. For PaddlePaddle model represented
             as a Python object, you can specify outputs as a PaddlePaddle Python objects or
-            a list of such objects.
+            a list of such objects. Example with ovc.convert_model: output=["out_1", "out_2"].
```

Comment:
Don't refer to `convert_model` as `ovc.convert_model`. It is exposed to user as `openvino.convert_model`. Just use `convert_model`. In this case, change is not needed at all.

Reply:
Also move example with a list before PaddlePaddle case description.